src/view/com/profile/ProfileHeader.tsx

```diff
@@ -433,8 +437,8 @@ let ProfileHeaderLoaded = ({
 
   const blockHide =
     !isMe && (profile.viewer?.blocking || profile.viewer?.blockedBy)
-  const following = formatCount(profile.followsCount || 0)
-  const followers = formatCount(profile.followersCount || 0)
+  const following = FormatCount(profile.followsCount || 0)
+  const followers = FormatCount(profile.followersCount || 0)
```

Comment:
Why was this function capitlalized? Usually that is reserved for React Components.

Reply:
I resolve the linting error. It reverted to lowercase.